README.md

```diff
@@ -41,8 +41,12 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
-```js
-// write code here
+```
+  for (var i = 0; i < foods.length; i++) {
```

Comment:
Close! What we were going for here, is just a console.log of each fruit. So the following might be an appropriate answer:

```
for(var i = 0; i < foods[0].length; i++){
  console.log("I want to eat a " + foods[0][i]);
}
```